src/main/java/de/tum/in/www1/exerciseapp/web/rest/ResultResource.java

```diff
@@ -300,7 +320,7 @@ public ResultResource(ResultRepository resultRepository, Optional<LtiService> lt
      * @return the ResponseEntity with status 200 (OK)
      */
     @DeleteMapping("/results/{id}")
-    @PreAuthorize("hasAnyRole('TA', 'ADMIN')")
+    @PreAuthorize("hasAnyRole('TA', 'INSTRUCTOR', 'ADMIN')")
     @Timed
     public ResponseEntity<Void> deleteResult(@PathVariable Long id) {
```

Comment:
This should only be possible for TAs and instrcutors of the corresponding exercise (and admins)

Reply:
Same. I believe we have no feature to delete results from user interface.

Reply:
Done.